core-services/prow/02_config/_boskos.yaml

```diff
@@ -341,7 +341,8 @@ resources:
   - tenant-cluster-3
   state: free
   type: kubevirt-quota-slice
-- names:
+- max-count: 8
```

Comment:
the min/max-count setting is mutually exclusive with the `names` setting (I am surprised that this doesn't trip the validation)

Reply:
OK then this is definitely the wrong approach then.
Do you know why there are fewer leases in https://grafana-prow-monitoring.apps.ci.l2s4.p1.openshiftapps.com/d/628a36ebd9ef30d67e28576a5d5201fd/boskos-dashboard?orgId=1 then there are listed in the https://github.com/openshift/release/pull/14398/files#diff-5169f2a74d1497f38a44e9adc57f6993269a89c3ddf90ab01f5d1d114ef61e58

Reply:
That graph is running clusters by platform, i.E. the number of leases that are currently in use, not the total number of leases. The latter is eight as expected:
```
$ k get resources|rg libvirt-ppc64le|wc -l
8
```

Reply:
I'll send you a slack w/ an image of what I mean, but I was talking about the right-hand size grid, which lists out the min, max, and current. The graph looks like it only cares about the current.